src/extension/background-script/actions/webln/signMessageOrPrompt.ts

```diff
@@ -1,18 +1,21 @@
 import utils from "~/common/lib/utils";
 import { Message } from "~/types";
 
+import { SignMessageResponse } from "../../connectors/connector.interface";
+
 //import db from "../../db";
 //import signMessage from "../ln/signMessage";
 
 const signMessageOrPrompt = async (message: Message) => {
   const messageToSign = message.args.message;
+  const isWebBTC = message.args.webbtc as boolean;
```

Comment:
we should not need this. 

any such special cases that need some `if` are code smells and add lots of complexity (even though it might look simple)

webbtc should completely sit on the side and not require any other code changes.